From 98cb9a95ce536d21be390934bd42dc2abde5959f Mon Sep 17 00:00:00 2001
From: Robert Yang <liezhi.yang@windriver.com>
Date: Thu, 29 Jan 2015 22:18:06 -0800
Subject: [PATCH] Makefile: fix for parallel build

Fixed parallel issue:
* btreplay.o: file not recognized: File truncated
  collect2: error: ld returned 1 exit status

* btreplay/btreplay.c:47:18: fatal error: list.h: No such file or directory
  #include "list.h"
                  ^
  compilation terminated.

  Don't make btreplay twice will fix the problem

Upstream-Status: Pending

Signed-off-by: Robert Yang <liezhi.yang@windriver.com>
---
 Makefile |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index fe35d02..e7b1e94 100644
--- a/Makefile
+++ b/Makefile
@@ -17,8 +17,8 @@ btt/btt:
 btreplay/btrecord:
 	$(MAKE) -C btreplay
 
-btreplay/btreplay:
-	$(MAKE) -C btreplay
+btreplay/btreplay: btreplay/btrecord
+	echo Building $@
 
 %.o: %.c
 	$(CC) -o $*.o -c $(ALL_CFLAGS) $<
-- 
1.7.9.5

